#run SortmeRNA on merged fastq files
for ((sample=1;sample<=16;sample++));do
bsub -q long -G team152 -J sortmerna.${sample} -n1 -R "span[hosts=1] select[mem>10000] rusage[mem=10000]" -M10000 -o /lustre/scratch115/projects/paxgene/logs/sortmerna.${sample}.out -e /lustre/scratch115/projects/paxgene/logs/sortmerna.${sample}.err /software/team152/sortmerna-2.1-linux-64/./sortmerna --ref /software/team152/sortmerna-2.1-linux-64/rRNA_databases/silva-bac-16s-id90.fasta,/software/team152/sortmerna-2.1-linux-64/index/silva-bac-16s-db:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/silva-bac-23s-id98.fasta,/software/team152/sortmerna-2.1-linux-64/index/silva-bac-23s-db:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/silva-arc-16s-id95.fasta,/software/team152/sortmerna-2.1-linux-64/index/silva-arc-16s-db:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/silva-arc-23s-id98.fasta,/software/team152/sortmerna-2.1-linux-64/index/silva-arc-23s-db:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/silva-euk-18s-id95.fasta,/software/team152/sortmerna-2.1-linux-64/index/silva-euk-18s-db:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/silva-euk-28s-id98.fasta,/software/team152/sortmerna-2.1-linux-64/index/silva-euk-28s:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/rfam-5s-database-id98.fasta,/software/team152/sortmerna-2.1-linux-64/index/rfam-5s-db:/software/team152/sortmerna-2.1-linux-64/rRNA_databases/rfam-5.8s-database-id98.fasta,/software/team152/sortmerna-2.1-linux-64/index/rfam-5.8s-db --reads /lustre/scratch115/projects/paxgene/fastqFiles/21121_${sample}.merged.fastq  --num_alignments 1 --fastx --aligned /lustre/scratch115/projects/paxgene/fastqFiles/pastSortMeRNA/21121_${sample}.merged.fastq_rRNA --other /lustre/scratch115/projects/paxgene/fastqFiles/pastSortMeRNA/21121_${sample}.merged.fastq_non_rRNA --log -v --paired_in
done
